src/components/TeaserFront/Tile.js

```diff
@@ -162,32 +177,40 @@ const Tile = ({ children, attributes, image, alt, onClick, color, bgColor, align
     IMAGE_SIZE.large,
     false
   )
+  let containerStyle = {
+    background,
+    cursor: onClick ? 'pointer' : 'default',
+    justifyContent
+  }
+  if (imageOnly) {
+    containerStyle = {
+      ...containerStyle,
+      padding: 0
+    }
+  }
+  const imageStyle = imageOnly ? {maxHeight: '100%', maxWidth: '100%'} : {}
 
   return (
     <div
       {...attributes}
       {...styles.container}
       onClick={onClick}
-      style={{
-        background,
-        cursor: onClick ? 'pointer' : 'default',
-        justifyContent
-      }}
+      style={containerStyle}
       className='tile'
     >
       {imageProps && (
-        <div {...styles.imageContainer}>
+        <div {...(imageOnly ? styles.imageOnlyContainer : styles.imageContainer)}>
           <LazyLoad visible={aboveTheFold}>
             <img src={imageProps.src} srcSet={imageProps.srcSet} alt={alt}
-              {...styles.image} />
+              {...styles.image} style={imageStyle}/>
```

Comment:
Use an `imageOnly` css style here too. Feels odd to use a different pattern for the image container vs image.

Reply:
The reason for this pattern was specificity (style attribute is more specific than the tilegroup's parent selectors affecting maxwidth/maxheight). I'm using a consistent pattern now, at the cost of introducing !importants.